styles/src/base_layers.ts

```diff
@@ -66,19 +66,15 @@ export function nolabels_layers(
       filter: [
         "in",
         "kind",
-        "national_park",
         "park",
         "cemetery",
-        "protected_area",
-        "nature_reserve",
-        "forest",
         "golf_course",
         "wood",
-        "nature_reserve",
         "forest",
         "scrub",
         "grassland",
         "grass",
+        "glacier",
```

Comment:
We forgot this one in the past. Now high-zoom glaciers are visible again. Maybe I should add a render test for this...

Reply:
Yes, I think the render test is most useful for this class of issue, but do we have enough data in the CI sample data?